src/Bridge/Symfony/Routing/IriConverter.php

```diff
@@ -116,9 +116,14 @@ public function getItemFromIri(string $iri, array $context = [])
     /**
      * {@inheritdoc}
      */
-    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH): string
+    public function getIriFromItem($item, int $referenceType = UrlGeneratorInterface::ABS_PATH/**, array $context = []**/): string
```

Comment:
This class is final, so you can add the parameter. You should add a new `ContextAwareDataConverterInterface` however